### PR TITLE
Fix electron package

### PR DESCRIPTION
**Pull request by campersau**
use latest electron packager and build them in CI.

fixes https://github.com/FredrikNoren/ungit/issues/593 for the second time :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,15 +25,18 @@ addons:
...
   - if [ "$GIT_VERSION" = "edge" ]; then sudo add-apt-repository ppa:git-core/ppa -y && sudo apt-get update -q && sudo apt-get install -y git; fi
   - npm ci
 before_script:
+  - wine --version
   - git config --global user.email "test@testy.com"
   - git config --global user.name "Test testy"
   - git version
   - npm run build
 after_success:
+  - DISPLAY= npm run package
```
Comment (campersau) [inline .travis.yml]: This took me 1 1/2 days to find out 🤦‍♂

**Comment by aitboudad** [comment]
@campersau I'm unable to use copy and paste when using electron app, is it intended?

**Comment by campersau** [comment]
@aitboudad can you please open a new issue?
Also please include the ungit version and os because for me copy & paste from and to ungit is working with ungit electron 1.5.0 on windows 10.

**Comment by aitboudad** [comment]
done #1251

**Pull request merged**